Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -50,30 +50,36 @@ struct ClientTabsSearchWrapper {
     var tab: RemoteTab
 }
 
+struct SearchViewModel {
+    let isPrivate: Bool
+    let isBottomSearchBar: Bool
+    // TODO: Move logic in view model
```

Comment:
```suggestion
```

Do you still need this todo?

Reply:
It's like a future todo 🥲 I don't wanna go over the code of the SearchViewController in this PR - but there's definitely some separation of concerns we could do in there. Next time we play in that area we can do it. I just left that there as a reminder

Reply:
Task: https://github.com/mozilla-mobile/firefox-ios/issues/9977